tests/utils/terms.test.ts

```diff
@@ -0,0 +1,13 @@
+import en from 'src/locales/en.json'
+import strings from '../../src/locales/strings'
+
+describe('Terms', () => {
+  it('should test if terms are in sync', () => {
+    try {
+      expect(Object.keys(en).length).toBe(Object.keys(strings).length)
```

Comment:
maybe worth checking the two sets are equal? the usual case is something got added or removed though I suppose